insolar/payload/payload.go

```diff
@@ -139,6 +141,14 @@ func (h *MessageHash) IsZero() bool {
 	return true
 }
 
+func (m RequestInfo) RequestAsObjectReference() *insolar.Reference {
```

Comment:
Can we move these to logicrunner helpers? Doesn't have to be a method.

Reply:
Both of these functions? `RequestAsObjectReference`+`RequestReference`?

Reply:
Yep. Let's not pollute the record package.

Reply:
kk, pushed